pkg/v1/tkg/tkgctl/create_cluster_test.go

```diff
@@ -554,10 +558,70 @@ var _ = Describe("Unit tests for - (Azure) - cluster_azure.yaml as input file fo
 	})
 })
 
+var _ = Describe("Unit tests for - TKGS Cluster - cluster_tkgs.yaml as input file for 'tanzu cluster create -f cluster_tkgs.yaml' use case", func() {
```

Comment:
nit: describe at high level what the tests do, but "Unit tests for -" is kinda redundant.